src/modules/checkIsValidUserInput.js

```
@@ -9,7 +9,25 @@
  * @return {boolean} - True if the user input is valid, false otherwise
  */
 function checkIsValidUserInput(userInput) {
-  /* Write your code here */
+  if (userInput.length !== 4) {
+    return false;
+  }
+
+  if (!/^\d{4}$/.test(userInput)) {
+    return false;
```

Issue: The function returns false if the input is not all digits. It should throw an error with a descriptive message instead.